benchmarks/sequel/benchmark.rb

```diff
@@ -1,5 +1,5 @@
 require "harness"
-require 'random/formatter'
+require "securerandom" # Provides `Random::Formatter` in Ruby 2.6+
```

Comment:
Just want to make sure. We want to make sure that execution is deterministic as much as possible. Does securerandom need to be seeded separately from the default Ruby RNG?

Reply:
Just need to make sure that the random alphanumeric values are the same from one execution to the next, and if not, check if there is a way to fix a seed in `harness/harness-common.rb`

Reply:
Another option would be to define our own `Random.alphanumeric` in `harness-common.rb`, which we could then use to yank the `random/formatter` and `securerandom` dependencies from this benchmark and also the activerecord benchmark.